cmd/dump.go

```diff
@@ -21,6 +21,7 @@ import (
 	"code.gitea.io/gitea/modules/util"
 
 	"gitea.com/go-chi/session"
+
```

Comment:
unnecessary blank line.

Reply:
That's necessary, they belongs to different domains in my mind.